server.js

```diff
@@ -181,33 +181,33 @@ async function getAccountsAndTransactions(userId, res) {
   // GET Accounts
   const accounts = await getAccounts(consumerApiPath, userId, accessToken);
 
-  accounts.forEach(account => {
+  for (const account of accounts) {
```

Comment:
might wanna try using `.forEach` instead
```suggestion
  accounts.forEach(account => {
```

Reply:
I'm not certain that I _can_ have this as a `.forEach` instead of a `.for`.

I originally had this as a `.forEach` but got an error from Node about the `await getTransactions(consumerApiPath, userId, accountId, accessToken)` call 'not being within an async function' (paraphrasing the error message).

After some research, the [accepted answer and conversation on Stack Overflow](https://stackoverflow.com/questions/37576685/using-async-await-with-a-foreach-loop) made sense to me as the reason why this can't be a `.forEach`, although I admit I could be wrong.

Reply:
👍 I was wondering if there was some gotcha I wasn't seeing. I'm cool with the old school for loop.